### ISSUE TITLE
plexapi.exceptions.BadRequest: (406) not_acceptable

### ISSUE THREAD

**Issue by UnR34L** [pkkid/python-plexapi#158]
#Plex Media Server Version : 1.7.6.4058
OS: FreeBSD 11.0-STABLE
Python: 2.7.13
pip 9.0.1

websocket-client and requests are installed.

```
#!/usr/local/bin/python2.7

from plexapi.server import PlexServer
baseurl = 'http://192.168.0.54:32400'
token = 'TOKEN'
plex = PlexServer(baseurl, token)

ozzy = plex.library.section('Trailers').get('Ozzy')
client = plex.client("DESKTOP-B88TLB1")
client.playMedia(ozzy)
```

**Traceback**

`Traceback (most recent call last):
  File "./test.py", line 10, in <module>
    client.playMedia(ozzy)
  File "/usr/local/lib/python2.7/site-packages/plexapi/client.py", line 441, in playMedia
    }, **params))
  File "/usr/local/lib/python2.7/site-packages/plexapi/client.py", line 181, in sendCommand
    return self.query(key, headers=headers)
  File "/usr/local/lib/python2.7/site-packages/plexapi/client.py", line 153, in query
    raise BadRequest('(%s) %s; %s' % (response.status_code, codename, errtext))
plexapi.exceptions.BadRequest: (406) not_acceptable;`

**Comment by Hellowlol** [comment]
What is the client your trying to play on?

**Comment by UnR34L** [comment]
With the list of clients I get from 

for client in plex.clients():
print(client.title)


I tried 'DESKTOP-B88TLB1' which should be Plex Media Player then I tried 'Plex Web (Chrome)' and a Samsung Smart TV but that gives the error 

`plexapi.exceptions.BadRequest: (401) unauthorized; <html><head><title>Unauthorized</title></head><body><h1>401 Unauthorized</h1></body></html>`

It says unauthorized so I tried the logging into MyPlex method, which gave the same error, and I know the username and password are correct.

**Comment by UnR34L** [comment]
Tried a Android client, it plays but gives the error

`Traceback (most recent call last):
  File "./test.py", line 9, in <module>
    client.playMedia(alien)
  File "/usr/local/lib/python2.7/site-packages/plexapi/client.py", line 441, in playMedia
    }, **params))
  File "/usr/local/lib/python2.7/site-packages/plexapi/client.py", line 181, in sendCommand
    return self.query(key, headers=headers)
  File "/usr/local/lib/python2.7/site-packages/plexapi/client.py", line 155, in query
    return ElementTree.fromstring(data) if data.strip() else None
  File "/usr/local/lib/python2.7/xml/etree/ElementTree.py", line 1311, in XML
    parser.feed(text)
  File "/usr/local/lib/python2.7/xml/etree/ElementTree.py", line 1653, in feed
    self._raiseerror(v)
  File "/usr/local/lib/python2.7/xml/etree/ElementTree.py", line 1517, in _raiseerror
    raise err
xml.etree.ElementTree.ParseError: syntax error: line 1, column 0
`

To sum up, keeping the script consistent and only changing the client.

**plexapi.exceptions.BadRequest: (406) not_acceptable**
Plex Media Player

**401 Unauthorized**
Plex Web (Chrome)
Samsung Smart TV

**xml.etree.ElementTree.ParseError: syntax error: line 1, column 0**
Android Client (but will play file)

**Comment by Hellowlol** [comment]
Ill try to debug this today.

**Comment by JonnyWong16** [comment]
I asked the PlexTogether guys. You need to send a subscribe command before the `playMedia` command.

```
self.sendCommand('timeline/subscribe', **{'port': 32400, 'protocol': 'http', 'X-Plex-Device-Name': 'Python-PlexAPI'})
```

Then in the `playMedia` command also add the `X-Plex-Device-Name`:
https://github.com/pkkid/python-plexapi/blob/master/plexapi/client.py#L433-L441
```
        self.sendCommand('playback/playMedia', **dict({
            'machineIdentifier': self._server.machineIdentifier,
            'address': server_url[1].strip('/'),
            'port': server_url[-1],
            'offset': offset,
            'key': media.key,
            'token': media._server._token,
            'containerKey': '/playQueues/%s?window=100&own=1' % playqueue.playQueueID,
            'X-Plex-Device-Name': 'testing',
        }, **params))
```

**Comment by Hellowlol** [comment]
@UnR34L Can you try https://github.com/pkkid/python-plexapi/pull/160 on samsung etc.

**Comment by UnR34L** [comment]
@Hellowlol Fixed!

I'll Open a new issue for the '401 Unauthorized' error.

**Comment by JonnyWong16** [comment]
I'm pretty sure `playMedia` won't work on Plex Web (no remote control capabilities). The PlexTogether guys couldn't get it to work.

**Comment by pkkid** [comment]
PlayMedia works on Plex web. You just have to have Plex web open with the real ip (not Plex.tv), and I believe you need to be on the same network as well.

**Comment by Hellowlol** [comment]
Dunno why but i cant even see chrome when calling clients.
@pkkid good to see you back. You got prs to review. :D

**Comment by JonnyWong16** [comment]
Just give @hellowlol write access. :P

**Comment by Hellowlol** [comment]
I believe i do. I just like getting stuff reviewed.

**Comment by JonnyWong16** [comment]
Ah, yes. Accessing Plex Web using the server IP does allow the `playMedia` command.

**Comment by pkkid** [comment]
I'm around, I've been watching all the checkins to keep up with whats going on. Thanks so much for being active on these bug reports! Give me another few days to catch up on some things and I'll try to get back into the full swing. After this next batch of changes I think we really need to update the pip version (and possibly automate it as you suggested).

**Comment by platers** [comment]
I am having this issue as well. Is the pip updated with the fixed version yet? If not is there a way I can implement the changes myself?

**Comment by Hellowlol** [comment]
Just install from github
